### DISCUSSION TITLE
Track dependents count for legacy package names

### DISCUSSION THREAD

**Discussion by spalladino** [community/community#23338] [New to GitHub]
<p>Hey there! A few months ago, we have renamed some of our npm packages. Since we are particularly interested in keeping track of the Dependents count for them, we have added the package.json files with the older names to our repos, so that we can still track them by changing from the new to the old name in the dropdown, as seen in the picture below, corresponding to <a href="https://github.com/OpenZeppelin/openzeppelin-test-helpers/network/dependents?package_id=UGFja2FnZS0yMzY2MTcxNTU%3D" rel="nofollow noopener">this repo</a> (with the legacy package.json <a href="https://github.com/OpenZeppelin/openzeppelin-test-helpers/blob/master/legacy/package.json" rel="nofollow noopener">here</a>, corresponding to the <a href="https://www.npmjs.com/package/openzeppelin-test-helpers" rel="nofollow noopener">old package name</a>).</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/e5052b18c87b43f12dc6527aafff42dd374df583.png" data-download-href="/uploads/short-url/wG0pETCB4vNSoCPuBzKPelkY76P.png?dl=1" title="Screenshot from 2020-04-01 10-23-44.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/429604/181076196-40cbc316-d2fa-49b1-885d-f43c1ac619d7.png" alt="Screenshot from 2020-04-01 10-23-44.png" data-base62-sha1="wG0pETCB4vNSoCPuBzKPelkY76P" width="690" height="259" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/e5052b18c87b43f12dc6527aafff42dd374df583_2_690x259.png, https://aws1.discourse-cdn.com/github/original/1X/e5052b18c87b43f12dc6527aafff42dd374df583.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/e5052b18c87b43f12dc6527aafff42dd374df583.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/e5052b18c87b43f12dc6527aafff42dd374df583_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2020-04-01 10-23-44.png</span><span class="informations">770×290 24.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Now, this has not worked in <a href="https://github.com/OpenZeppelin/openzeppelin-sdk" rel="nofollow noopener">another of our repositories</a> (openzeppelin-sdk). This repo holds several packages, of which two are public and have been both renamed. The new names (<code>@openzeppelin/cli</code> and <code>@openzeppelin/upgrades</code>) are <a href="https://github.com/OpenZeppelin/openzeppelin-sdk/network/dependents?package_id=UGFja2FnZS00NjU2MTU0OTU%3D" rel="nofollow noopener">correctly tracked</a>, but the <a href="https://github.com/OpenZeppelin/openzeppelin-sdk/tree/master/legacy" rel="nofollow noopener">legacy names</a> (zos and zos-lib) are not, even though the package.json files for them are present in the repo using the same trick.</p>
<p>Any idea what we can do to start tracking dependents for zos and zos-lib in the openzeppelin-sdk repository? Thanks a lot!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/spalladino">@spalladino</a>,</p>
<p>Thank you for being here! I’ve now manually assigned those packages to the OpenZeppelin/openzeppelin-sdk repo. Give it a few and you should be able to see it now.</p>

**Comment by spalladino** [comment]
<p>It works like a charm! Thanks so much Andrea!!</p>
